html/pfappserver/root/static.alt/src/components/pfFieldAction.vue

```diff
@@ -0,0 +1,299 @@
+<template>
+  <b-form-row class="pf-field-action mx-0 mb-1 px-0 col-12" align-v="center"
+    v-on="forwardListeners"
+  >
+    <b-col v-if="$slots.prepend" cols="1" align-self="start" class="pt-1 text-center col-form-label">
+      <slot name="prepend"></slot>
+    </b-col>
+    <b-col cols="4" align-self="start">
+
+      <pf-form-chosen
+        v-model="localType"
+        ref="localType"
+        label="text"
+        track-by="value"
+        :placeholder="typeLabel"
+        :options="fields"
+        :vuelidate="typeVuelidateModel"
+        :invalid-feedback="typeInvalidFeedback"
+        class="mr-1"
+        collapse-object
+      ></pf-form-chosen>
+
+    </b-col>
+    <b-col cols="6" align-self="start" class="pl-1">
```

Comment:
The `value` inputs need more space than the `type` values, otherwise the pfFormDatetime input (with moments) gets compressed too much so that it hides and masks the full input value. Cols pf 4/6 are needed to avoid this.